data_steward/utils/bq.py

```diff
@@ -211,19 +211,21 @@ def delete_dataset(project_id,
 
 def is_validation_dataset_id(dataset_id):
     """
-    Checks if dataset_id is a validation dataset
+    Check if  bq_consts.VALIDATION_PREFIX is in the dataset_id
+
+    :param dataset_id:
 
-    :param dataset_id: identifies the dataset
```

Comment:
This patch is removing valid documentation strings.  Did you mean to do this?

Reply:
Can you elaborate?

Reply:
your patch removed line 216 in pink which at least has a vague description of the dataset_id parameter.  it adds `:param dataset_id:`, with no description at all.  Did you mean to remove this description?

Reply:
No, I didn't mean to remove it. However, since it was initially vague, should I change it to be more descriptive?

Reply:
That's not exactly what this parameter is doing.  It is not validating the dataset.  It is checking if the validation prefix is in the dataset name.

Reply:
I changed the description of `dataset_id` to say `ID (name) of the dataset to validate` should I change it again?